examples/morpheusvm/controller/action_indexer.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package controller
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/extension/indexer"
+)
+
+var _ indexer.SuccessfulTxSubscriber = (*actionHandler)(nil)
+
+type actionHandler struct {
```

Comment:
Should we call this something else...? Something related metrics or something?

Reply:
acceptedActionMetricsHandler? This is a bit too long imo and I slightly prefer actionHandler in this case unless you have an alternative suggestion.